src/styles/desktop/components/watch_page.scss

```diff
@@ -73,7 +73,7 @@ $MARGIN_HORIZONTAL_COUNTDOWN: 10vw;
     width: 100vw;
 
     #show-date-text {
-      font-size: 4.44vw;
+      font-size: 4px;
```

Comment:
Almost sure this isn't a change you wanted to make...

![image](https://user-images.githubusercontent.com/12106484/109425120-d1734a00-79b4-11eb-8472-c51acb7c076c.png)